lib/ansible/executor/module_common.py

```diff
@@ -640,10 +683,16 @@ def recursive_finder(name, data, py_module_names, py_module_cache, zf):
     unprocessed_py_module_names = normalized_modules.difference(py_module_names)
 
     for py_module_name in unprocessed_py_module_names:
+        # HACK: this probably works, but is not guaranteed to; need a more explicit flag?
+        if not py_module_cache[py_module_name][1].startswith('/'):
+            dir_prefix = ''
```

Comment:
Whats the scenario that this is HACK'ing around? Is it when 'ansible_collection.*' imports are used in module code?

Reply:
Basically it's a way to identify a collection-based import as opposed to an old module_utils import